content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -61,6 +61,20 @@ among the comma-delimited, ordered list of values for the `--enable-admission-pl
 the API server component. For more information on API server command-line flags,
 check [kube-apiserver](/docs/admin/kube-apiserver/) documentation.
 
+Claims can also be created without any StorageClass while no default exists. In that case
+their StorageClass will remain unset until a default becomes available, then the claim
+will get the new default assigned by `RetroactiveDefaultStorageClass` [feature](/docs/concepts/storage/persistent-volumes/#retroactive-storageclass-assignment).
+
+### Retroactive StorageClass assignment
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+A PVC can be created without specifying any StorageClass. This can happen either if there is no default StorageClass set by the administrator or [`DefaultStorageClass` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass) is disabled. PVs with `StorageClassName` set to `""` will continue to bind to such PVCs until default StorageClass is created.
+
+When default StorageClass becomes available any existing PVCs without `StorageClassName` will get this value updated retroactively to the default StorageClass name. In order to keep binding to PVs with `StorageClassName` set to `""` (while default StorageClass is present) users need to set PVC `StorageClassName` to `""`.
+
+This behavior helps administrators change default storage class by removing the old one first and then creating or setting another one. This brief window while there is no default causes PVCs without `StorageClassName` created at that time to not have any default, but due to the retroactive StorageClass assignment this way of changing defaults is safe.
```

Comment:
```suggestion
This behavior helps administrators change default storage class by removing the old one first and then creating or setting another one. This brief window while there is no default causes PVCs with an empty `storageClassName` created at that time to not have any default, but due to the retroactive StorageClass assignment this way of changing defaults is safe.
```

Reply:
Same as above, I intentionally used the word "without" because it means that the value is set to nil (meaning there's a key in yaml file but value is left out), or the key is not there.

That's why I mention `""` around L74 to explain it. I can only think of adding a sentence that would say that "without" means no key or no value to make it more clear.

Reply:
We need to clarify this - I really didn't understand the sense of the existing document.

It helps a lot to use the active voice to explain what actions the control plane takes, and then follow that up with a higher-level explanation of what that means for the cluster operator.

Reply:
Ack. I've changed it be more clear on the high level and with each paragraph it gets more specific - up to the point where it explains the differences between no value, no key, and empty string value. 

@sftim PTAL and let me know if the current version is ok.